### ISSUE TITLE
@conda-forge-admin, please update team

### ISSUE THREAD

**Issue by zklaus** [conda-forge/esmvalcore-feedstock#23]
The last team update was not successful, presumably due to the ongoing azure problems. See #17.

**Comment by valeriupredoi** [comment]
good call, Klaus, I thought since the CODEOWNERS and meta files did not contain Manu, I should add him there, but failed to realize that in fact you already added him in the `rc` branch (sorry if you had false flag notifications from me) - could it be he needs to be there but on `master`, to be able to do admin-y stuff?

**Comment by zklaus** [comment]
Actually, I did add him in master, as you can easily check.

@schlunma, you should have received an invitation to join the conda-forge organization, perhaps a separate one for the team. Did you? Have you checked your spam-folder or your github notifications in case you don't receive them via email?

**Comment by schlunma** [comment]
No, unfortunately I didn't get an e-mail about that :disappointed:

**Comment by zklaus** [comment]
@conda-forge-admin, please update team

**Comment by valeriupredoi** [comment]
@zklaus cheers, man - not in the [README](https://github.com/conda-forge/esmvalcore-feedstock/blob/master/README.md) though, and we should remove Javi (sorry to say it, but we should :cry: )

**Comment by zklaus** [comment]
Ah, but that's the point: Those changes will be done by the bot once the team is updated. This should not be done manually because a bunch of things have to be coordinated: README.md, CODEOWNERS, recipe.yaml all live in the repository, but for the permissions etc membership in orgas and teams are important which are not directly controlled from within the repo.

**Comment by valeriupredoi** [comment]
ah had no idea, thought it was DIY all round :hammer_and_wrench:

**Comment by zklaus** [comment]
Closing this issue since it is not needed anymore: The bot will either have sent out the invitation or not, but this issue doesn't matter for that.
